Neos.Neos/Configuration/Settings.yaml

```diff
@@ -66,15 +66,6 @@ Neos:
           collapsed: true
 
     userInterface:
-      # Sets the route after login. There will be one argument "node" set with the lastVisited or site node (and correct context)
-      # The below configuration sets this to the "old" UI route.
-      routeAfterLogin:
-        values:
-          '@package': 'Neos.Neos'
-          '@controller': 'Frontend\Node'
-          '@action': 'show'
-          '@format': 'html'
-
```

Comment:
This should not be removed completely – the code still uses it (if set), so this must stay as the "definition" and documentation for the setting.

Reply:
Good catch @kdambekalns. The goal is still to get rid of it. Some more refactoring is needed though.

Reply:
Done. Haven't found any usages anymore.